inc/class-saml.php

```diff
@@ -89,13 +91,22 @@ class SAML {
 	 */
 	private $samlSettings = [];
 
+	/**
+	 * @var Log
+	 */
+	private $log;
+
 	/**
 	 * @return SAML
 	 */
 	static public function init() {
 		if ( is_null( self::$instance ) ) {
 			$admin = Admin::init();
-			self::$instance = new self( $admin );
+			$log = null;
+			if ( CloudWatchProvider::areEnvironmentVariablesPresent() ) {
+				$log = new Log( new CloudWatchProvider( 90, 'pressbooks-logs', 'pressbooks-plugin', 'saml-logs' ) );
```

Comment:
We could set those configuration from const values or even better from Env variables. As it is pretty temporary use, I will leave those values there.